packages/core/contracts/financial-templates/funding-rate-store/implementation/OptimisticOracle.sol

```diff
@@ -0,0 +1,357 @@
+// SPDX-License-Identifier: AGPL-3.0-only
+pragma solidity ^0.6.0;
+pragma experimental ABIEncoderV2;
+
+import "@openzeppelin/contracts/token/ERC20/IERC20.sol";
+import "@openzeppelin/contracts/token/ERC20/SafeERC20.sol";
+import "@openzeppelin/contracts/math/SafeMath.sol";
+
+import "../../../oracle/interfaces/StoreInterface.sol";
+import "../../../oracle/interfaces/OracleInterface.sol";
+import "../../../oracle/interfaces/FinderInterface.sol";
+import "../../../oracle/interfaces/IdentifierWhitelistInterface.sol";
+import "../../../oracle/implementation/Constants.sol";
+
+import "../../../common/implementation/Testable.sol";
+import "../../../common/implementation/Lockable.sol";
+import "../../../common/implementation/FixedPoint.sol";
```

Comment:
The only place you use `FixedPoint` is `payOracleFeesErc20`. Would it make sense to replace the use of `int` in this contract with a `FixedPoint.Signed`, seeing we have this functionality now?

Reply:
I was actually thinking that there's no reason to use FixedPoint in this class since it doesn't do any arithmetic. I figured it would be simpler just to stick with primitives. Thoughts?